packages/lab/src/static-list/StaticListItem.css

```diff
@@ -0,0 +1,18 @@
+.saltStaticListItem {
+  list-style: none;
```

Comment:
This will get reset by all: unset;

Also it's probably not needed since display: flex will remove the "discs"